### DISCUSSION TITLE
Where can I view all the property name used for ${property-name} ?

### DISCUSSION THREAD

**Discussion by 1000Fear** [mpv-player/mpv#14796] [Q&A] [answered]
I set `title=${chapter}` in osc.conf, it shows like this: ↓↓↓

![f6c95ee34986899b6acb9d9ac0b25f2e](https://github.com/user-attachments/assets/d483011c-7608-4903-9159-8da335738403)

I want it to display like this ↓↓↓, just like I move the cursor to seekbar, without a chapter number, What should I do?

![4e048c6f57675a58fc4d923f9621d1a4](https://github.com/user-attachments/assets/cd057e21-a341-4b47-a03d-1cc00b7aeb28)

**Comment by Argon-** [comment, marked as answer]
Here's a list: https://mpv.io/manual/master/#property-list
You could try `chapter-metadata/by-key/title`.

**Reply by 1000Fear** [reply]
Exactly what I need, thank you!

**Comment by hooke007** [comment]
See also https://mpv.io/manual/master/#options-list-properties

**Reply by 1000Fear** [reply]
Thank you too, I'll give it a try.
